### PR TITLE
test(sliderBehavior): add description and unit tests

### PR DESCRIPTION
**Pull request by kolaps33**

### CODE REVIEW

**Review by silviuaavram — APPROVED** [review]

**Review by jurokapsiar — APPROVED** [review]

File: packages/react/src/lib/accessibility/Behaviors/Slider/sliderBehavior.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+/**
+ * @description
+ * Adds attribute 'aria-disabled=true' based on the property 'disabled'.
+ * Adds attribute 'aria-orientation=vertical' or 'horizontal' based on the property 'vertical'.
+ * Adds attribute 'aria-valuemin' with value based on the property 'min'.
+ * Adds attribute 'aria-valuemax' with value based on the property 'max'.
+ * Adds attribute 'aria-valuenow' with value based on the property 'value'.
+ * Adds attribute 'aria-valuetext' with value based on the callback function 'getA11yValueMessageOnChange'.
+ */
+
```
Comment (layershifter) [inline packages/react/src/lib/accessibility/Behaviors/Slider/sliderBehavior.ts]:
```suggestion

```

nit

File: packages/react/test/specs/behaviors/sliderBehavior-test.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { sliderBehavior } from 'src/lib/accessibility'
+
+function generatePropertyTest(
+  propName: string,
+  propValue: string | boolean | number,
+  slot: string,
+  expectedAttr: string,
+  customExpResult?: string,
+) {
```
Comment (layershifter) [inline packages/react/test/specs/behaviors/sliderBehavior-test.ts]: Can we use object instead? It's hard to follow all function params

```tsx
generatePropertyTest('disabled', true, 'root', 'aria-disabled')
```
**vs**
```tsx
generatePropertyTest({ 
  propName: 'disabled',
  propValue: true, 
  slot: 'root',
  attr: 'aria-disabled'
})
```

**Review by layershifter — APPROVED** [review]
